content/recipes/prisma.md

```diff
@@ -391,6 +391,21 @@ export class PostService {
 
 Your `UserService` and `PostService` currently wrap the CRUD queries that are available in Prisma Client. In a real world application, the service would also be the place to add business logic to your application. For example, you could have a method called `updatePassword` inside the `UserService` that would be responsible for updating the password of a user.
 
+We need to register the new services with Nest so that it can perform the injection. We do this by editing our module file (`app.module.ts`) and adding the new services to the `providers` array of the `@Module()` decorator.
+
+```typescript
```

Comment:
I'm not sure if we need to show any code for this here as registering providers is pretty standard in nestjs apps

Reply:
@micalevisk I followed along the recipe, tried to run the app and ran into the error. What do you think about just mentioning it and leaving out the code?

Reply:
yeah we can have that mention to recall the dev, but the code itself is redundant here

Reply:
I adapted the change accordingly.